connectors/aws/aws-sqs/src/main/java/io/camunda/connector/inbound/model/SqsInboundQueueProperties.java

```diff
@@ -6,108 +6,41 @@
-    this.region = region;
-  }
-
-  public String getUrl() {
-    return url;
-  }
-
-  public void setUrl(final String url) {
-    this.url = url;
-  }
-
-  public boolean isContainAttributeNames() {
-    return attributeNames != null && !attributeNames.isEmpty();
-  }
-
-  public boolean isContainMessageAttributeNames() {
-    return messageAttributeNames != null && !messageAttributeNames.isEmpty();
-  }
-
-  public List<String> getAttributeNames() {
-    return attributeNames;
-  }
-
-  public void setAttributeNames(final List<String> attributeNames) {
-    this.attributeNames = attributeNames;
-  }
-
-  public List<String> getMessageAttributeNames() {
-    return messageAttributeNames;
-  }
-
-  public void setMessageAttributeNames(final List<String> messageAttributeNames) {
-    this.messageAttributeNames = messageAttributeNames;
-  }
-
-  public String getPollingWaitTime() {
-    return pollingWaitTime;
-  }
-
-  public void setPollingWaitTime(final String pollingWaitTime) {
-    this.pollingWaitTime = pollingWaitTime;
-  }
-
-  @Override
-  public boolean equals(final Object o) {
-    if (this == o) {
-      return true;
-    }
-    if (o == null || getClass() != o.getClass()) {
-      return false;
-    }
-    final SqsInboundQueueProperties that = (SqsInboundQueueProperties) o;
-    return Objects.equals(region, that.region)
-        && Objects.equals(url, that.url)
-        && Objects.equals(attributeNames, that.attributeNames)
-        && Objects.equals(messageAttributeNames, that.messageAttributeNames)
-        && Objects.equals(pollingWaitTime, that.pollingWaitTime);
-  }
-
-  @Override
-  public int hashCode() {
-    return Objects.hash(region, url, attributeNames, messageAttributeNames, pollingWaitTime);
-  }
-
-  @Override
-  public String toString() {
-    return "SqsInboundQueueProperties{"
-        + "region='"
-        + region
-        + "'"
-        + ", url='"
-        + url
-        + "'"
-        + ", attributeNames="
-        + attributeNames
-        + ", messageAttributeNames="
-        + messageAttributeNames
-        + ", pollingWaitTime='"
-        + pollingWaitTime
-        + "'"
-        + "}";
-  }
-}
+public record SqsInboundQueueProperties(
+    @Deprecated @TemplateProperty(ignore = true) String region,
+    @TemplateProperty(
+            id = "queue.url",
+            label = "Queue URL",
+            group = "queueProperties",
+            description = "Specify the URL of the SQS queue where you would like to subscribe to")
+        @NotBlank
+        String url,
+    @TemplateProperty(
+            id = "queue.attributeNames",
+            label = "Attribute names",
+            group = "input",
+            description =
+                "Array of queue attribute names. See <a href=\"https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/amazon-sqs/?amazonsqs=inbound\" target=\"_blank\">documentation</a> for details")
+        List<String> attributeNames,
```

Comment:
Do we accept FEEL expressions here? Consider changing the default as it will be `disabled` for inbound templates automatically when we implement https://github.com/camunda/connectors/issues/2282